### PR TITLE
22146 - Fixed the CO-OP Incorporation Review Page Bug

### PR DESCRIPTION
**Pull request by ArwenQin**
*Issue #:* /bcgov/entity#22146

*Description of changes:*
- Fixed the bug that: Unable to file a CO-OP fling when logged in as a staff user
  By fixing the Review and Confirm page bug. (Court Order section validity)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-730-v4kp6kj9.web.app

**Comment by ArwenQin** [comment]
Try: 
https://business-create-dev--pr-730-v4kp6kj9.web.app/incorporation-review-confirm?id=TXbxhqTMM1&accountid=3040

File: src/views/Incorporation/IncorporationReviewConfirm.vue [hunk 1]
```diff
@@ -315,6 +315,10 @@ export default class IncorporationReviewConfirm extends Vue {
 
   /** Is true when the certify conditions are not met. */
   get isCertifyInvalid (): boolean {
+    // Set the Court Order section valid, when it is not visible
+    if (!(this.isBaseCompany && this.isRoleStaff)) {
+      this.setCourtOrderValidity(true)
+    }
```
Comment (severinbeauvais) [inline src/views/Incorporation/IncorporationReviewConfirm.vue:321]: Having a computed (getter) set a value in the store is counter-intuitive.

Try a solution here instead:
https://github.com/bcgov/business-create-ui/blob/0388fcd7ae1f1b618b184677856ae23ae432679f/src/store/store.ts#L775

Comment (ArwenQin) [reply src/views/Incorporation/IncorporationReviewConfirm.vue]: Updated

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-730-v4kp6kj9.web.app

**Comment by ArwenQin** [comment]
Try:
https://business-create-dev--pr-730-v4kp6kj9.web.app/incorporation-review-confirm?id=TXbxhqTMM1&accountid=3040

File: src/views/Incorporation/IncorporationReviewConfirm.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/views/Incorporation/IncorporationReviewConfirm.vue]: LGTM.

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**